pydisdrometer/plot/plot.py

```diff
@@ -32,14 +32,18 @@ def plot_dsd(dsd, range=None, log_scale=True, tighten=True):
     fig_handle: Figure Handle
 
     '''
-
-    fig_handle = plt.figure()
+    ax =  parse_ax(ax)
+    fig = parse_ax(fig)
 
     colors = [('white')] + [(cm.jet(i)) for i in range(1, 256)]
```

Comment:
This breaks on mine. You're using range in argument list which is overwriting the python reserved keyword causing this to see the range function as NoneType